src/main/java/org/folio/linked/data/configuration/KafkaListenerConfiguration.java

```diff
@@ -53,4 +54,11 @@ public ConsumerFactory<String, DataImportEvent> dataImportEventConsumerFactory()
     return new DefaultKafkaConsumerFactory<>(config, new StringDeserializer(), deserializer);
   }
 
+  @Bean
+  public RetryTemplate retryTemplate(FolioKafkaProperties folioKafkaProperties) {
+    return RetryTemplate.builder()
+      .maxAttempts((int) folioKafkaProperties.getRetryDeliveryAttempts())
```

Comment:
minor
suggest to add null check with default value

Reply:
Updated code to remove this Bean configuration